.github/workflows/comment_pr.yml

```diff
@@ -0,0 +1,25 @@
+name: Comment code metrics report to PR
+
+on:
+  pull_request:
+    types: [opened]
```

Comment:
Ah, I understand. Why did you limit this to only `opened`?

Reply:
@kenji-miyake Because the bot doesn't need to comment every time we push.
The URL doesn't change, so it's enough to comment only when the PR opens.

Reply:
Oh, exactly!